src/features/MoviesBrowser/MoviePage/MovieInfo/index.js

```diff
@@ -24,28 +25,38 @@ const MovieInfo = ({
   rate,
   votes,
   description,
-}) => (
-  <Main>
-    <Wrapper>
-      <Image src={poster} />
-      <DataContainer>
-        <Title>{title}</Title>
-        <Year>{productionYear}</Year>
-        <AdditionalData>
-          Production: <Text>{productionPlace}</Text>
-        </AdditionalData>
-        <AdditionalData>
-          Release date: <Text>{date}</Text>
-        </AdditionalData>
-        <TagsWrapper>
-          <Tag>{tags}</Tag>
-          {/* add map method later */}
-        </TagsWrapper>
-        <Rating rate={rate} votes={votes} type="info" />
-      </DataContainer>
-      <MovieDescription>{description}</MovieDescription>
-    </Wrapper>
-  </Main>
-);
+}) => {
+  const isMedia = useMediaQuery({ maxWidth: 767 });
+
+  return (
+    <Main>
+      <Wrapper>
+        <Image src={poster} />
+        <DataContainer>
+          <Title>{title}</Title>
+          <Year>{productionYear}</Year>
+          <AdditionalData>
+            <div>
+              {isMedia ? "" : "Production: "}
+              <Text>{productionPlace}</Text>
+            </div>
+            <div>
+              {isMedia ? "" : "Release date: "}
+              <Text>{date}</Text>
+            </div>
```

Comment:
to discussion - which format of Date we want : dd/mm/yyyy (as at Figma) or mm/dd/yyyy (in my opinion more accurate, as we are working for en/us Client, am I right ? :D :D )

Reply:
I believe Figma option is more optimal in terms of making the project exactly as client requested 👌

Reply:
okey, so You need to change format you used ;)